x-pack/plugins/cases/public/components/edit_connector/index.tsx

```diff
@@ -372,16 +335,32 @@ export const EditConnector = React.memo(
                 </EuiFlexGroup>
               </EuiFlexItem>
             )}
-            {pushCallouts == null &&
+            {!hasErrorMessages &&
               !isLoading &&
               !editConnector &&
-              permissions.push &&
+              hasPushPermissions &&
               actionsReadCapabilities && (
                 <EuiFlexItem data-test-subj="has-data-to-push-button" grow={false}>
-                  <span>{pushButton}</span>
+                  <span>
+                    <PushButton
```

Comment:
`usePushToService` does not return any more components. I created separate components in separate files.